src/gtimelog/gtimelog.css

```diff
@@ -1 +1,5 @@
-/* We don't have any custom CSS today! */
+/* GTK+ themes other than Adwaita ignore the 'monospace' property and
+   use a proportional font for text widgets. */
+#log_view {
+	font-family: monospace;
```

Comment:
That's the wrong widget.  We want the _report_ view to be monospace (it's the one what aligns time columns with spaces).  The _log_ view uses tabstops for alignment and is fine with a proportional font.

Also, I was kind of hoping to see an example of how to use style classes (.monospace maybe?) since I've never used them before, but it's not a hard requirement.

Reply:
> That's the wrong widget. We want the _report_ view to be monospace (it's the one what aligns time columns with spaces). The _log_ view uses tabstops for alignment and is fine with a proportional font.
🫣 Sorry. Next try in the amended fix :roll_eyes: 
> 
> Also, I was kind of hoping to see an example of how to use style classes (.monospace maybe?) since I've never used them before, but it's not a hard requirement.
Good point. I used the "monospace" class in the amended commit.